granulate_utils/linux/cgroups/cgroup.py

```diff
@@ -73,28 +95,230 @@ def find_v2_hierarchy(resolve_host_root_links: bool = True) -> Optional[str]:
     if len(cgroup2_mounts) > 1:
         raise Exception("More than one cgroup2 mount found!")
     path = cgroup2_mounts[0].mount_point
-    if resolve_host_root_links:
-        path = ns.resolve_host_root_links(path)
+    path = ns.resolve_host_root_links(path)
     return path
 
 
-def get_cgroup_mount(controller: str, resolve_host_root_links: bool = True) -> Optional[str]:
+# Cgroup v1 and v2 handle the cgroup assigned processes using cgroup.procs
+CGROUP_PROCS_FILE = "cgroup.procs"
+
+
+class CgroupCore:
+    """
+    This object handles the general operations for Cgroup management:
+    hierarchy management, controllers management, assigning processes.
+    cgroup_full_path: The full path for this CGroupCore object
```

Comment:
> The full path for this CGroupCore object

what does this mean?

Reply:
I changed the description a bit as it was too amorphic I think.
`cgroup_full_path` is `cgroup_absolute_path` - I guess I'll rename it.